src/main.py

```diff
@@ -130,12 +130,14 @@ def order_subscription():
 @login_required
 def order_packets():
     form = OrderPacketsForm()
+    if current_user.user_type == manifest.USER_TYPES.ADMIN:
+        form.email.choices = [(email, email) for email in db.get_all_emails()]
     descriptions = [(manifest.PACKETS[name]["name"], manifest.PACKETS[name]["description"]) for name in
                     manifest.PACKETS]
     descriptions = dict(descriptions)
 
     if form.validate_on_submit():
-        user = db.get_client_by_email(form.email.data)
+        user = db.get_client_by_email(form.email.data if form.email.data is not None else current_user.email)
```

Comment:
Are these checks necessary? Is it possible that the form will return None email?

Reply:
Form will return None in case when non admin users send filled form. Then current user's email is used as default

Reply:
I can't order any packets. This call returns None. Is it supposed to work like that? I tried regisitering, loging out and in, and still got the crash. Similar case to editing user subscription.